# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../doxygen/doxywizard-flex-bug.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- doxygen-1.3.9/src/config.l.orig	2004-10-12 11:26:04.676878072 +0200
+++ doxygen-1.3.9/src/config.l	2004-10-12 11:27:04.547776312 +0200
@@ -34,6 +34,10 @@
 
 #include "lang_cfg.h"
 
+#ifdef __cplusplus
+extern "C" int isatty(int fd);
+#endif
+
 #undef Config_getString
 #undef Config_getInt
 #undef Config_getList

